content/blog/2025-02-03-submit_posit_conf/index.en.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Tips for Submitting a Video Abstract to Posit Conf 2025"
+author: 
+  - Nic Crane
+  - Sayantika Banik
+date: '2025-02-03'
+---
+
+Thinking about submitting a talk for **Posit Conf 2025** but feeling nervous about recording a video abstract? You're not alone! It can feel intimidating at first, but the good news is—it's just **one minute** and all about showcasing your idea effectively.
+
+## **Tips from the R-Ladies Global Team**
+
+We know that recording a video can feel like a challenge, so we asked members of **R-Ladies Global** to share their best advice:
```

Comment:
do the tips have to be from the global team or can they be from the R-Ladies community more broadly?

Reply:
Good point, updated on other PR